### PR TITLE
Updated text-functionality.c

### PR DESCRIPTION
**Pull request by Anurag-Modak**
uppdated load_text_from_file function

### Checklist:

- [ ] My code has been run through [clang-format](https://github.com/obsproject/obs-studio/blob/master/.clang-format).
- [ ] I have read the [**contributing** document](https://github.com/obsproject/obs-studio/blob/master/CONTRIBUTING.rst).
- [ ] My code is not on the master branch.
- [ ] The code has been tested.
- [ ] All commit messages are properly formatted and commits squashed where appropriate.
- [ ] I have included updates to all appropriate documentation.

### CODE REVIEW

**Review by derrod — CHANGES_REQUESTED** [review]
- Fill out the PR template
- Fix the formatting 
- Fix the commit message

**Comment by Anurag-Modak** [comment]
> If you are interested in working on this please read the previous discussions on what is actually required to fix the issue, [#7595 (comment)](https://github.com/obsproject/obs-studio/issues/7595#issuecomment-1977012523)

Can you elaborate the issue to me

File: plugins/text-freetype2/text-functionality.c [hunk 1]
```diff
@@ -390,60 +390,64 @@ static void remove_cr(wchar_t *source)
...
+        // File is already in UTF-16 format
+        if (srcdata->text != NULL) {
+            bfree(srcdata->text);
+            srcdata->text = NULL;
+        }
+        srcdata->text = bzalloc(filesize);
+        bytes_read = fread(srcdata->text, filesize - 2, 1, tmp_file);
+
+        // Ensure null-termination of the string
+        srcdata->text[(filesize - 2) / sizeof(wchar_t)] = L'\0';
```
Comment (norihiro) [inline plugins/text-freetype2/text-functionality.c:425]: `bzalloc` initialize the allocated region with 0.
Why was this line added?

**Comment by kkartaltepe** [comment]
Thanks for the interest in the issue, but im going to close this since I have gone ahead to implement the fix I was intending.

**Pull request closed without merging**